src/test/kotlin/no/nav/syfo/testhelper/generator/DialogmeldingFraBehandlerGenerator.kt

```diff
@@ -89,18 +89,52 @@ val fellesformatXML = """<?xml version="1.0" ?>
     <MottakenhetBlokk avsender="12312341" avsenderFnrFraDigSignatur="${UserConstants.BEHANDLER_PERSONIDENT.value}" avsenderRef="SERIALNUMBER=996871045, CN=LEGEHUSET NOVA DA, O=LEGEHUSET NOVA DA, C=NO" ebAction="Henvendelse" ebRole="Sykmelder" ebService="HenvendelseFraLege" ebXMLSamtaleId="615356d4-f5e6-4138-a868-bbb63bd6195d" ediLoggId="1901162157lege21826.1" herIdentifikator="" meldingsType="xml" mottattDatotid="2019-01-16T21:57:43" partnerReferanse="${UserConstants.PARTNERID}" />
 </EI_fellesformat>"""
 
-fun generateDialogmeldingFraBehandlerDTO(
+fun generateDialogmeldingFraBehandlerDialogNotatDTO(
+    uuid: UUID = UUID.randomUUID(),
+    personIdent: PersonIdent = UserConstants.ARBEIDSTAKER_PERSONIDENT,
+    conversationRef: String = UUID.randomUUID().toString(),
+    antallVedlegg: Int = 0,
+) = KafkaDialogmeldingFraBehandlerDTO(
+    msgId = uuid.toString(),
+    msgType = DialogmeldingType.DIALOG_NOTAT.name,
+    navLogId = "1234asd123",
+    mottattTidspunkt = LocalDateTime.now(),
+    conversationRef = conversationRef,
+    parentRef = UUID.randomUUID().toString(),
+    personIdentPasient = personIdent.value,
+    personIdentBehandler = UserConstants.BEHANDLER_PERSONIDENT.value,
+    legekontorOrgNr = "987654321",
+    legekontorHerId = "",
+    legekontorOrgName = "",
+    legehpr = UserConstants.HPRID.toString(),
+    fellesformatXML = fellesformatXML,
```

Comment:
Den xml'en som kommer med her stemmer ikke helt overens med meldingen, så kanskje best å fikse det.

Reply:
Okey, hva er forskjellen? Litt usikker på hvor jeg skal lete i disse XMLene 🤔

Reply:
Ja, dette skal være xml'en som kommer utenfra, så den må stemme overens med dto'en (som inneholder noen av feltene fra xml'en). Bør nok være en egen xml per generert dto-objekt.

Reply:
Ja, at man feks kunne laget en funksjon i stedet for å generere XML, som tar inn feks dialogmeldingstype (`DIALOG_NOTAT` eller `DIALOG_SVAR`) og litt andre typer felter? Kan evt fikse det i en egen PR 👍🏼 Det er jo testdata så sånn sett ikke kritisk for denne PRen, men fint å ha riktig.

Har vi noen eksempler andre steder for hvordan en sånn XML ser ut? `padm2`?

Reply:
Ligger noen i `isyfomock` og antagelig `isdialogmelding` også

Reply:
Kan ikke se at den XMLen som ligger her stemmer med ForesporselSvar heller. Den har jo type DIALOG_NOTAT og mangler Document-element (som er der forskjellene ligger). Fint å få rydde opp i det, men kan godt gjøres i egen PR tenker jeg.